src/test/java/org/apache/commons/crypto/cipher/AbstractCipherTest.java

```diff
@@ -220,9 +218,7 @@ private void byteBufferTest(final String transformation, final byte[] key, final
 				final byte[] b = new byte[output.remaining()];
 				output.get(b);
 				final byte[] c = new byte[encResult.remaining()];
-				encResult.get(c);
-				Assert.fail("AES failed encryption - expected " + new String(DatatypeConverter.printHexBinary(b))
-						+ " got " + new String(DatatypeConverter.printHexBinary(c)));
+				assertThrows(IllegalArgumentException.class, () -> encResult.get(c));
```

Comment:
Please preserve the assertion message, it will be handy when the test fails.

Reply:
Done